.travis.yml

```diff
@@ -14,7 +14,7 @@
 
 language: go
 go:
-- 1.16
+- 1.18
```

Comment:
The generated container from the Dockerfile s still using go-toolset:1.13... is it ok to run the linters with 1.18 while the compiled version will be 1.13?
Note: Go-toolset latest version is 1.16

Reply:
We actually have to due to changes in golinter